.github/workflows/publish-bufs.yaml

```diff
@@ -90,3 +90,50 @@ jobs:
         message: 'chore(proto-ts): bump version'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  dart:
+    needs: check-preconditions
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - name: Clone Repo
+      uses: actions/checkout@v3
+    - name: Install tools
+      run: |
+        pip install yq
+        wget -O /usr/local/bin/semver https://raw.githubusercontent.com/fsaintjacques/semver-tool/master/src/semver
+        chmod +x /usr/local/bin/semver
+    - uses: dart-lang/setup-dart@v1
+    - name: 'Set up Cloud SDK'
+      uses: 'google-github-actions/setup-gcloud@v1'
+      with:
+        version: '>= 363.0.0'
+    - name: Auth Service Account
+      env:
+        KEY_FILE: ${{ secrets.PUB_DEV_KEY }}
+      run: |
+        echo "$KEY_FILE" > key-file.json
```

Comment:
Is this really necessary? We are already using `google-github-actions/setup-gcloud` in this workflow, shouldn't that be able to do the job?. Take a look at https://github.com/helpwave/web/blob/main/.github/workflows/cicd-tasks.yaml#L128.